v1/python-sdk/tutorials/automl-with-azureml/forecasting-many-models/auto-ml-forecasting-many-models.ipynb

```diff
@@ -379,8 +379,16 @@
    "source": [
     "### Set up training parameters\n",
     "\n",
-    "This dictionary defines the AutoML and many models settings. For this forecasting task we need to define several settings inncluding the name of the time column, the maximum forecast horizon, and the partition column name definition.\n",
+    "We need to provide ``ForecastingParameters`` and ``AutoMLConfig`` objects. For the forecasting task we also need to define several settings including the name of the time column, the maximum forecast horizon, and the partition column name definition.\n",
```

Comment:
Please add the `ManyModelsTrainParameters` object to the list that user needs to provide.